pageserver/src/http/models.rs

```diff
@@ -140,6 +128,12 @@ pub struct LocalTimelineInfo {
     pub current_logical_size: Option<usize>, // is None when timeline is Unloaded
     pub current_logical_size_non_incremental: Option<usize>,
     pub timeline_state: LocalTimelineState,
+
+    pub wal_source_connstr: Option<String>,
+    #[serde_as(as = "Option<DisplayFromStr>")]
+    pub last_received_msg_lsn: Option<Lsn>,
+    /// the timestamp (in microseconds) of the last received message
```

Comment:
Not related to your PR and no need to change now, but while we at it, let's agree whether this HTTP timeline info API is a human-readable one or not. 

We do not use walreceiver info anywhere so far, AFAIK, so we can change it fluently still.